package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "wallet-ui",
```

Comment:
Could we change this to `@etherspot/wallet-ui` please? This brings the package names in line with our other developer toolkits

Reply:
suggested changes done